web/src/js/utils.js

```diff
@@ -81,7 +81,21 @@ function getCookie(name) {
 }
 const xsrf = `_xsrf=${getCookie("_xsrf")}`;
 
+
 export function fetchApi(url, options={}) {
+    if (global.MITMWEB_STATIC) {
+        let path = url.split('/'),
```

Comment:
Same here. Let's add ".json" to wherever `fetchApi` is called. :) No need to do it within here.